src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -64,9 +64,13 @@ public sealed class VSSolutionManager : IVsSolutionManager, IVsSelectionEvents,
         private readonly INuGetFeatureFlagService _featureFlagService;
         private readonly Microsoft.VisualStudio.Threading.AsyncLazy<DTE> _dte;
         private readonly Microsoft.VisualStudio.Threading.AsyncLazy<IVsSolution> _asyncVSSolution;
+        private readonly Microsoft.VisualStudio.Threading.AsyncLazy<IVsBackgroundSolution> _asyncVSBackgroundSolution;
```

Comment:
Is this a 17.5 API? I can't find it in index at all.

Reply:
yes, it's new in 17.5. There's no problems finding it, or the service implementation, using AzDO code search.